static/app/views/profiling/profileSummary/content.tsx

```diff
@@ -0,0 +1,91 @@
+import {useMemo} from 'react';
+import styled from '@emotion/styled';
+import {Location} from 'history';
+
+import {SectionHeading} from 'sentry/components/charts/styles';
+import * as Layout from 'sentry/components/layouts/thirds';
+import Pagination from 'sentry/components/pagination';
+import {FunctionsTable} from 'sentry/components/profiling/functionsTable';
+import {ProfilesTable} from 'sentry/components/profiling/profilesTable';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {PageFilters, Project} from 'sentry/types';
+import {useFunctions} from 'sentry/utils/profiling/hooks/useFunctions';
+import {useProfiles} from 'sentry/utils/profiling/hooks/useProfiles';
+import {decodeScalar} from 'sentry/utils/queryString';
+
+const PROFILES_COLUMN_ORDER = [
+  'failed' as const,
+  'id' as const,
+  'timestamp' as const,
+  'device_model' as const,
+  'device_classification' as const,
+  'trace_duration_ms' as const,
+];
```

Comment:
```suggestion
const PROFILES_COLUMN_ORDER = [
  'failed',
  'id',
  'timestamp',
  'device_model',
  'device_classification',
  'trace_duration_ms',
] as const;
```

I assume you tried this right? Could we instead maybe type as keyof SomeColumns or something? The as const for each value seems redundant